CONTRIBUTING.md

```diff
@@ -0,0 +1,133 @@
+   1. Review the notebook for understandability.
+      1. Does the notebook clearly describe the processes happening in each cell? Does it provide context?
+   1. Review the code to ensure it's doing what it's supposed to.
+   1. Check for spelling errors.
+
+## Merging
+For Admins:
+1. If the notebook looks good, the reviewer will [merge the PR](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/merging-a-pull-request).
+1. This kicks off a series of [GitHub Actions](https://github.com/features/actions).
+   1. You can see the workflows at https://github.com/ioos/notebooks_demos/actions 
+1. Once the workflows are complete, check the [IOOS CodeLab](https://ioos.github.io/ioos_code_lab/) for the new notebook.
+1. Review the notebook for any display issues when presented on the web. Check for the extra cell at the bottom of the notebook. 
+   1. If there are issues, submit a new PR to resolve the issues.
+
+## Announcing
+When new notebooks, or significant changes to the site, are merged and pushed out, follow this procedure:
+1. **Admin** Move the appropriate Project card to [Announcement](https://github.com/orgs/ioos/projects/1#column-13186308). 
+1. Draft announcement text.
+   1. Include a brief summary of the new release.
+   1. Include any relevant links to the demo center.
+1. Distribute text through the following channels.
+   1. IOOS bi-weekly
+   1. EOTO
+   1. ioos_tech mail list
+   1. dmac mail list
+   1. Twitter/FB.
+   1. Appropriate slack channels
+
+## Logging bugs or changes
+1. Bugs should be reported through tickets in this repository. Be sure to include the notebook where the problem is found. Include a clear description of the issue or change that is needed.
+
+## Managing IOOS Data Demo Center Project Board
+The [project board](https://github.com/orgs/ioos/projects/1) captures the current progress on notebooks and adjustments to the IOOS Data Demo Center.
+
+| **Column name** | **Purpose** |
+------------------|--------------
+Icebox | Back burner tickets to keep in the fray.
+Backlog | Tickets to be worked on next.
+Working | Tickets actively being contributed to.
+Needs Review | Tickets where a PR has been requested to merge.
+Announcement | Tickets that have been merged and an announcement needs to be made.
+Done | All completed tickets.
+
+## Local management of notebooks_demo
+This section walks through one way to manage the notebooks_demo git repository on your local machine.
+
+Clone your fork of the notebooks_demo repository to a known location on your machine: 
+
+```$ git clone https://github.com/MathewBiddle/ioos_code_lab.git```
+
+Set the upstream branch to the ioos/notebooks_demos repository:
+
+```$ git remote add upstream https://github.com/ioos/ioos_code_lab.git```
+
+Keep your local master up-to-date with the upstream master ([walkthough](https://stefanbauer.me/articles/how-to-keep-your-git-fork-up-to-date)):
+
+```
+$ git fetch upstream
+$ git pull upstream/master master
```

Comment:
This repo is using `master` b/c we imported from the old one but new repositories are set to `main` instead. Should we take this opportunity to change this repository from `master` to `main` before folks start forking it?

Reply:
I don't fully understand the complexities of `master` vs `main` (other then they are different names), so I lean on you for advice here. If that's the recommendation moving forward, then lets follow that.

I know I've been renaming new repo's from `main` back to `master` because that's what I use my git commands and things get all messed up if I use `main`. With that being said, maybe my practice should change with this change.

Reply:
Yes, let's change from `master` to `main`. I did some research.